hubs.yaml

```diff
@@ -112,7 +112,7 @@ hubs:
               - choldgraf
               - ericvd-ucb
 - name: pfw
-    org_name: Purdue Fort Wayne
```

Comment:
I think the issue is that `org_name` has a different indentation level from `name` (same for the following keys). THey all need to be at the same level